shim.js

```diff
@@ -0,0 +1 @@
+process.browser = true;
```

Comment:
this isn't a shim anymore, so maybe just set `process.browser = true` directly in App.js now?

Reply:
:man_shrugging: happy to remove that.

Reply:
Copying it in Apps throws
>undefined is not and object (evaluating process.version.split)

Note renaming it after discussion because it still is a black magic shim.